sdk/identity/identity/samples/AzureCacheForRedis/ioredis.md

```diff
@@ -0,0 +1,273 @@
+## Azure Cache for Redis: Azure AD with ioredis client library
+
+### Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Authenticate with Azure AD - Hello World](#authenticate-with-azure-ad-ioredis-hello-world)
+- [Authenticate with Azure AD - Handle Reauthentication](#authenticate-with-azure-ad-handle-reauthentication)
+- [Authenticate with Azure AD - Using Token Cache](#authenticate-with-azure-ad-using-token-cache)
```

Comment:
I really thought that "using token cache" would be about `@azure/identity-cache-persistence`, but it's about locally storing the token in a variable (which you already do in the reauthentication sample as well, so I don't understand the token cache sample).

I think we should avoid this phrasing of "token cache" because it sounds like it's going to use the plugin.